app/controllers/orders_controller.rb

```diff
@@ -97,19 +97,20 @@ def checkout
 
   def payment
     stripe_session = StripeHandler.new(success_order_url, cancel_order_url, @order).create_stripe_session
-    @order.payments.create(session_id: stripe_session.id, currency: stripe_session.currency, status: 'Pending', total_amount_in_cents: @order.net_in_cents)
+    @order.payments.create(session_id: stripe_session.id, currency: stripe_session.currency, status: 'pending', total_amount_in_cents: @order.net_in_cents)
     redirect_to stripe_session.url, allow_other_host: true
   end
 
   def success
-    @order.Placed!
-    @order.update_column(:order_at, Time.current)
-    @order.payments.last.Successful!
-    OrderMailer.received(current_user.orders.Placed.last).deliver_later
+    checkout_session = Stripe::Checkout::Session.retrieve(@order.payments.last.session_id)
```

Comment:
Ill suggest to create a service for stripe that has simple methods like `retrieve_session`, etc instead of we writing `Stripe::Checkout::Session` again and again

Reply:
done